src/app/transfer-bitclout/transfer-bitclout.component.html

```diff
@@ -90,7 +90,7 @@
 </button>
 
 <div class="p-15px d-flex flex-column align-items-center">
-  <p>Share to receive $CLOUT</p>
+  <p>Scan QR to Send {{ globalVars.loggedInUser.ProfileEntryResponse.Username }} $Clout</p>
```

Comment:
Can we do `Scan QR code to send {{ globalVars.loggedInUser.ProfileEntryResponse?.Username }} $CLOUT`?

Don't forget a user can be logged in and not have a `ProfileEntryResponse`

Reply:
Done :)